pyproject.toml

```diff
@@ -19,10 +20,15 @@ python = "^3.6"
 pydantic = "^1.7.4,!=1.8,!=1.8.1"
 structlog = "^20.1.0"
 colorama = {version = "^0.4.3", optional = true}
-
 # For Pydantic
 dataclasses = {version = "^0.7", python = "~3.6"}
 
+# For RTD poetry integration
+sphinx = {version = "^4.0.2", optional = true}
+m2r2 = {version = "^0.2.7", optional = true}
+sphinx-rtd-theme = {version = "^0.5.2", optional = true}
+toml = {version = "^0.10.2", optional = true}
```

Comment:
Do these need to be listed as core dependencies at all (even optional ones) or can they be kept only as dev dependencies?

Reply:
Discussed these are needed for poetry/pip integration into RTD.